styles.css

```diff
@@ -18,13 +18,13 @@ html {
 }
 
 h1 {
-    font-size: 8rem;
+    font-size: 10rem;
     position: relative;
     top: 40%;
 }
 
 h1 ~ p {
-    font-size:1.4rem;
+    font-size:2.3rem;
```

Comment:
Should be a space after the colon, for readability

There are a few of these, won't mark each one